dali/pipeline/pipeline.cc

```diff
@@ -486,6 +494,7 @@ void Pipeline::Build(std::vector<PipelineOutputDesc> output_descs) {
   vector<string> outputs;
   for (const auto &out_desc : output_descs_) {
     string name = out_desc.name;
+    // what device is it? Op placement, output placement, requested placement?
```

Comment:
Is this an open question or general note on uncertainty of life?

Reply:
I think it was the latter (but a bit of a former), removed.